plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFunctionFunctions.java

```diff
@@ -40,63 +40,48 @@ private FissionFunctionFunctions()
     }
 
     @ScalarFunction("fission_dnsdb")
-    @Description("Converts the string to alternating case")
+    @Description("Send get request to fission dnsdb endpoint")
     @SqlType(StandardTypes.BIGINT)
-    public static long dnsdb(@SqlType(StandardTypes.VARCHAR) Slice slice) throws JSONException, IOException
+    public static long FetchDnsDb(@SqlType(StandardTypes.VARCHAR) Slice slice) throws JSONException, IOException
```

Comment:
Method names should start with a lowercase letter
```suggestion
    public static long fetchDnsDb(@SqlType(StandardTypes.VARCHAR) Slice slice) throws JSONException, IOException
```